src/js/modules/accounts/components/ProfilePage.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { Link } from "react-router-dom";
+import injectSheet from "react-jss";
+
+import { SignOutForm } from "./forms";
+import { signOut } from "../actions";
+
+const styles = {
+  successMessage: {
+    color: "green",
+  },
+};
+
+const ProfilePage = ({ classes, session, signOut, status }) => {
```

Comment:
status is a little generic

Reply:
It's function is generic as well. Status holds the messages for all forms.

Reply:
I'd just expand the relevant fields, like error and form

Reply:
If status is a redux form you can decompose it

Reply:
It's still really generic though: message and error. Idt it's necessary.